build.gradle.kts

```diff
@@ -114,7 +114,7 @@ subprojects {
 
 buildscript {
     dependencies {
-        classpath("io.swagger.core.v3:swagger-gradle-plugin:2.1.13")
+        classpath("io.swagger.core.v3:swagger-gradle-plugin:2.2.2")
```

Comment:
isn't it possible to use the `${swagger}` variable here too?

Reply:
I was trying but it didn't worked, so I dived into the gradle thing and I got it working by defining the `swagger` variable again in the `dependencies` block :tada: